cabal-install/src/Distribution/Client/Init/Types.hs

```diff
@@ -406,6 +417,12 @@ popList = pop >>= \a -> case P.safeRead a of
     Nothing -> throwPrompt $ BreakException ("popList: " ++ show a)
     Just as -> return as
 
+checkInvalidPath :: String -> a -> PurePrompt a
+checkInvalidPath path act = 
+      if path `elem` ["."] then
```

Comment:
Do we expect to add more "invalid paths" to the list? :) It would be nice to document the intent so that a subsequent refactoring doesn't remove this line in favour of a direct match in the function head.

Reply:
Yes, that is the case, I shall add a comment stating such intent explicitly.

Reply:
Wonderful

Reply:
Done.